diplomarbeit/app.vue

```diff
@@ -24,9 +24,10 @@
          id="projektbeschreibung">
         Unser Projekt
       </p>
-      <p class="text-justify sm:mb-20 text-sm sm:text-xl leading-6 sm:leading-9 mx-8 sm:mx-[400px] max-w-[80ch]">
-        Im Rahmen dieser Diplomarbeit wird von uns eine Web-Applikation entwickelt, welche spielerisch die
-        Projektmanagementmethode Scrum vermitteln soll. Lernende bauen in Kleingruppen Städte aus Papier und erlangen
+      <p class="text-justify sm:mb-20 text-sm sm:text-xl leading-6 sm:leading-9 mx-8 sm:mx-[400px] max-w-[80ch] hyphens-manual">
+        ScrumpliCity entsteht im Rahmen einer Diplomarbeit an der HTL Rennweg. Als Projekt wird von uns eine
+        Web-Applikation entwickelt, welche spielerisch die
```

Comment:
```suggestion
        ScrumpliCity entsteht im Rahmen einer Diplomarbeit an der HTL Rennweg. Wir entwickeln eine
        Web-Anwendung, welche spielerisch die
```
"Als Projekt wird von uns ... entwickelt" klingt für mich komisch